api/serializers.py

```diff
@@ -53,10 +53,31 @@ class Meta:
         fields = "__all__"
 
 
-class InstructorRecordingsSerializer(serializers.ModelSerializer):
+class DynamicFieldsModelSerializer(serializers.ModelSerializer):
+    """
+    A ModelSerializer that takes an additional `fields` argument that
+    controls which fields should be displayed.
+    """
+
+    def __init__(self, *args, **kwargs):
+        # Don't pass the 'fields' arg up to the superclass
+        fields = kwargs.pop('fields', None)
+
+        # Instantiate the superclass normally
+        super(DynamicFieldsModelSerializer, self).__init__(*args, **kwargs)
+
+        if fields is not None:
+            # Drop any fields that are not specified in the `fields` argument.
+            allowed = set(fields)
+            existing = set(self.fields)
+            for field_name in existing - allowed:
+                self.fields.pop(field_name)
```

Comment:
I'm curious, why did you have to do this, @aaladaruncc? 

Wouldn't simply adding `transcript` give what you want?

Reply:
The reason I built this was so that we can select the specific fields within the serializer. Becuase we didn't want to include the instructor field according to the ticket on jira. I also thaought this would be better for future api's that request very specific information from an object. I can re implement this though if needed

Reply:
@aaladaruncc, it would be easier to simply remove the `instructor` field from the `InstructorRecordingsSerializer`